packages/wrangler/src/dev/miniflare.ts

```diff
@@ -952,6 +983,15 @@ export async function buildMiniflareOptions(
 		}
 	}
 
+	if (config.bindings.images && config.imagesLocalMode) {
+		if (!didWarnImagesLocalModeUsage) {
+			didWarnImagesLocalModeUsage = true;
+			logger.info(
+				"You are using Images local mode. This only supports resizing, rotating and transcoding."
+			);
+		}
+	}
```

Comment:
This will also need an equivalent warning for remote dev warning about usage charges (assuming there are usage charges)

Reply:
We don't have any usage charges for the preview endpoint - these requests are pretty cheap compared to e.g. Workers AI